tools/check-changelogger-use.php

```diff
@@ -65,7 +65,7 @@ function usage() {
 			usage();
 			break;
 		default:
-			if ( substr( $argv[ $i ], 0, 1 ) !== '-' ) {
+			if ( ! str_starts_with( $argv[ $i ], '-' ) ) {
```

Comment:
@anomiex Could I have your opinion on this? Since we all run PHP 8.2 in our dev environments, should this trigger an error here?

Reply:
Yeah, this should be fine since we want 8.2 for the tooling.

To make phpcs happy we'll want to add `<rule ref="Jetpack-Compat-82" />` into `tools/.phpcs.dir.xml` and `tools/.phpcs.dir.phpcompatibility.xml` alongside what's already there. Probably the same to `.github/.phpcs.dir.xml` and `.github/.phpcs.dir.phpcompatibility.xml` too.

There's a possible exception in that some scripts might want to be run from the Tests or Linting jobs that install older PHP to test/lint with them, but we should be able to catch those when they make the Tests or Linting jobs fail.

Reply:
Giving this a try in 5c07039bd87bf29e6b29d3029d0c90c2991b201b